README.md

```diff
@@ -1,24 +1,59 @@
+
+Once you finish this project, you will be considered a software team member and be expected to contribute towards AMP's codebase.
+
+## Instructions for the Project
+Note: This project assumes you know at least some basic git / version control. If you are stuck at any point in the project, feel free to reach out to the Software Leads for guidance.
+
+### Getting Started with a Workspace and Version Control
+1. Install [ROS Kinetic Kame](http://wiki.ros.org/kinetic) in a [Virtual Machine](https://www.vmware.com/products/workstation-player/workstation-player-evaluation.html) running [Ubuntu 16.04](https://releases.ubuntu.com/16.04/). (This may change as there are talks to port v1 to [Noetic](http://wiki.ros.org/noetic))
+2. Initialize a ROS catkin workspace in the home directory of the VM, and name it `intro-project`.
+3. Initialize git inside the project folder. Make sure that when you `ls`, you can see the `build/`, `devel/`, and `src/` directories.
+4. Checkout to a new branch which will contain all the files for your project. This can be done via the following command: 
+```bash
+git checkout -b solution-dev
+``` 
+5. Add and commit your initial workspace to this branch, you can name this commit `initial commit`.
+```bash
+git add . && git commit -m "initial commit"
+```
+
+### Linking your Project to a Remote Repository
+1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) this repository to your GitHub account, make sure the fork is public for code review.
+2. In your project, configure the remote repository by running the command:
+```bash
+git remote add origin <link-to-your-fork>
+```
+3. Setup an upstream link and push the commit with the command:
+```bash
+git push --set-upstream origin solution-dev
+```
+4. You can now work on the project and have it saved in a remote repository. This allows you to work on the project in multiple devices easily and protects your project from accidents!
+
+### Submitting your Project via a Pull Request
+1. Make sure the branch `solution-dev` in your fork of this repository has your project. Make sure all files are in the branch by running `git status`, it will tell you if there are any inconsistencies between your local project and the remote repository.
+2. In this Repository's website (on github.com), create a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) to merge your development branch into this repo's master brach. Add the software leads (GitHub users @zghera, @gfaout, and @ihagedo) as reviewers for your PR.
```

Comment:
```suggestion
2. In this Repository's website (on github.com), create a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) to merge your development branch into this repo's master branch. Add the GitHub users @zghera, @alanssitis, and @Reschivon as reviewers for your PR.
```
I think that since you and @Reschivon have the most recent hand's on exposure with this, you guys could help me out with onboarding new members rather than Gautam and Isaac for this round of newcomers. Does that sound alright to you guys?

Reply:
Sounds good to me. I have some experience on boarding new members from my old team.

Reply:
That makes sense to me!